index.html

```diff
@@ -0,0 +1,77 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital@1&display=swap" rel="stylesheet">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@3.2.0/fonts/remixicon.css" rel="stylesheet">
+    <link rel="stylesheet"
+    href="https://unpkg.com/boxicons@latest/css/boxicons.min.css">
+    <title>Bett Enock Portfolio</title>
+</head>
+<body>
+    <header class="icons">
+        <nav class="nav-container">
+            <div class="mylogo">
+                <a href="#"><img src="/Assets/images/cvimage2.JPG" alt="logo"></a>
+            </div>
+            
+            <ul class="nav-list">
+                <li> <a href="#" class="active">About</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+            <div class="menu_icons">
+                <a href="#"><img class="harmburgericon" src="/Assets/images/Harmburger Menu.svg" alt="Closeicon"></a>
+                <a href="#"><img class="closeicon" src="/Assets/images/Closeicon.svg" alt="Closeicon"></a>
+            </div>
+        </nav>
+    </header>
+
+
+    <section class="main-container">
+        <!-- Headline Section -->
+        <section class="main-container hero-container">
+            <article class="headline-container">
+                
+        
+                <div class="headline-content">
+                    <div class="headline-text">
+                        <hgroup>
+                            <h1 class="primary-text">Hey there. <br>I am Alex.</h1>
+                            <h2 class="secondary-text">I'm a software developer</h2>
+                            <p class="description-text">I can help you build a product , 
+                                feature or website Look through some of my work and experience! 
+                                If you like what you see and have a project you need coded, 
+                                do not hesitate to contact me.
+                            </p>
+                        </hgroup>
+    
+                    </div>
+                    
+                    <div class="socialmedia">
+                        <ul class="socialmedia-icons">
+                            <li><a href="#"><img src="/Assets/images/github.svg" alt="github"></a> </li>
+                            <li><a href="#"><img src="/Assets/images/Linkedin icon.svg" alt="Linkedin"></a> </li>
+                            <li><a href="#"><img src="/Assets/images/angel list.svg" alt="angellist"></a> </li>
+                            <li><a href="#"><img src="/Assets/images/medium.svg" alt="Medium"></a> </li>
+                        </ul>
+                    </div>
+                    <img class="background-image" src="/Assets/images/Illustration.svg" alt="header-banner">
```

Comment:
I suggest you work on the position of the background image so as to match the given design😉. currently, as a result of the current position of the background image, some of the texts are hard to see which can lead to a frustrating user experience😉. kindly ensure your background image is rightly positioned as shown in the design😉. try to consider the spacings around the image in the design👍.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="350" alt="image" src="https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/64643995/c928d87e-14d1-429c-bacf-a0e262cf85f3">|<img width="350" alt="image" src="https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/64643995/17c476ed-ee88-4221-89e3-9c55125b39b9">|